src/unix/lwt_main.mli

```diff
@@ -49,22 +49,20 @@ val yield : unit -> unit Lwt.t [@@deprecated "Use Lwt.pause instead"]
 
       @deprecated Since 5.5.0 [yield] is deprecated in favor of the more general
       {!Lwt.pause} in order to avoid discrepancies in resolution (see below) and
-      stay compatible with other execution environments such as js_of_ocaml.
+      stay compatible with other execution environments such as js_of_ocaml. *)
 
-      Currently, paused promises are resolved more frequently than yielded promises.
-      The difference is unintended but existing applications could depend on it.
-      Unifying the two pools of promises into one in the future would eliminate
-      possible discrepancies and simplify the code. *)
-
-val abandon_yielded_and_paused : unit -> unit
+val abandon_yielded_and_paused : unit -> unit [@@deprecated "Use Lwt.abandon_paused instead"]
 (** Causes promises created with {!Lwt.pause} and {!Lwt_main.yield} to remain
     forever pending.
 
     [yield] is now deprecated in favor of the more general {!Lwt.pause}.
     Once [yield] is phased out, this function will be deprecated as well.
```

Comment:
This line should be updated as well. We broke what we promised earlier, but maybe that's okay. :-)

Reply:
> We broke what we promised earlier, but maybe that's okay. :-)

What do you mean?

Reply:
Currently it says "Once `yield` is phased out, this function will be deprecated as well.", but `yield` has not been phased out yet.

Reply:
Yield is deprecated. I'm not sure what "phase out" means though. So maybe we should wait longer? I'm in favour of deprecating this.

Reply:
Oh, I guess that also makes sense. I was thinking about "phasing it out" as "removing it". Personally I'd love to clean this up as soon as possible regardless of what's written there. I guess we agree. :-)

Reply:
I updated the comment. It now simply mentions that yield is deprecated.